usefulUtilities/sitV2/appResources/appData/sitServerV2.js

```diff
@@ -0,0 +1,213 @@
+            _this.heartbeatInterval = Script.setInterval(function () {
+                if (DEBUG) {
+                    console.log("server heartbeat");
+                }
+                if (_this.isOccupied) {
+                    if (DEBUG) {
+                        console.log("server isOccupied");
+                    }
+                    Entities.callEntityClientMethod(
+                        _this.currentClientSessionID,
+                        _this.entityID,
+                        "check"
+                    );
+            
+                    // If the check call to the client script does not return checkResolved
+                    // Will open the chair to other avatars to sit
+                    Script.setTimeout(function () {
+                        if (_this.resolved === true) {
+                            // Seat is occupied
+                            _this.resolved = false;
+                        } else {
+                            // Seat is not occupied
+                            _this.isOccupied = false;
+                            _this.currentClientSessionID = false;
+                        }
+                    }, RESOLVED_TIMEOUT_TIME_MS);
+                }
+            }, HEARTBEAT_INTERVAL_TIME_MS);
+        }
+    }
+
+
+    // Remotely callable
+    // Called from client to open the chair to other avatars
+    function onStandUp() {
+        _this.isOccupied = false;
+        _this.currentClientSessionID = false;
+        if (_this.heartbeatInterval) {
+            Script.clearInterval(_this.heartbeatInterval);
+            _this.heartbeatInterval = false;
+        }
+    }
+
+
+    // Remotely callable
+    // Remove sittable local entities from every client in range passed in by sitClient
+    function removeAllOtherSittableOverlays(id, params) {
+        for (var i = 0; i < params.length; i++) {
+            Entities.callEntityClientMethod(
+                params[i],
+                _this.entityID,
+                "onLeaveCanSitZone"
+            );
+        }
+    }
+
+
+    // Remotely callable
+    // Add sittable local entities for this chair to every client in range passed in by sitClient
+    function addAllOtherSittableOverlays(id, params) {
+        if (DEBUG) {
+            console.log("ADD ALL OTHER SITTABLE OVERLAYS");
+        }
+        if (_this.isOccupied === false) {
+            for (var i = 0; i < params.length; i++) {
+                if (DEBUG) {
+                    console.log("avatar1");
+                }
+                Entities.callEntityClientMethod(
+                    params[i],
+                    _this.entityID,
+                    "onEnterCanSitZone"
+                );
+            }
+        }
+    }
+
+    // #region CAN SIT ZONE
+
+
+    // Create can sit zone
+    var CAN_SIT_M = 5;
+    function createCanSitZone() {
+        var properties = Entities.getEntityProperties(_this.entityID);
+        _this.canSitZoneID = Entities.addEntity({
```

Comment:
does this need to have `interactive: false` ?

Reply:
added